images/tags.md

```diff
@@ -38,3 +40,53 @@ To use an existing tag as the default tag, click the **vertical ellipses** for a
 tag and select **Make default**.
 
 ![Set Existing Tag as Default](../assets/existing-tag-as-default.png)
+
+## Image tag lifecycle
+
+Coder uses image tags to determine which image variant on which the user wants
```

Comment:
This is a great improvement over our existing documentation, thanks for taking this on Mike! I think we should distinguish between three things:

1. What Coder does with tags (the refresh behavior, default tag handling)
2. How we recommend users use tags (note: I don't think the way we're currently using images/tags is ideal, for reasons I'll describe in an RFC)
3. How we currently use tags for our enterprise-images

A general challenge with Docker is that tags act similarly to Git's branches as well as tags (or, in other words, a git tag that is force-pushed). This means that the meaning of tags is arbitrary and varies according to what authors decide to do, though there are common practices that become de-facto standards (which, we're not currently following)

Reply:
I tried to provide some examples... people typically understand "latest" but we do some OS flavor and some version specific... Just re-reading this the use of 2nd person isn't good and the examples are contrived. 

It would be better if we just made a suggestion for how to use image tags and then let folks know they'd can do whatever they need.